include/deal.II/base/data_out_base.h

```diff
@@ -1496,8 +1502,7 @@ namespace DataOutBase
     unsigned int node_dim;
 
     /**
-     * The number of vertices per cell. Equal to
-     * GeometryInfo<node_dim>::vertices_per_cell. We need to store
+     * The number of vertices per cell. We need to store
```

Comment:
I guess we should remove this variable and determine its value on-the-fly.

Reply:
I'm not sure how we can get the number of vertices inside the `n_cells()` method in that case! Any suggestion?

Reply:
This function looks odd^^ Let's keep it as it is right now and think about mixed meshes later. Could you add an assert somewhere that all cells have the same type?

Reply:
Done! ✔️